dev/wlp-jakartaee-transform/rules/jakarta-bundles.properties

```diff
@@ -15,3 +15,5 @@ com.ibm.ws.security.authorization.jacc.ejb=io.openliberty.security.authorization
 com.ibm.ws.security.authorization.jacc.web=io.openliberty.security.authorization.internal.jacc.web,,"Security Web support for JACC 2.0 Service","Security Web support for JACC 2.0 Service"
 com.ibm.websphere.appserver.api.jacc=io.openliberty.jacc.2.0,,"JACC Service Provider Interface","JACC Service Provider Interface, version 2.0"
 com.ibm.ws.security.authentication.filter=io.openliberty.security.authentication.internal.filter,,"Security Authentication filter","Security Authentication filter"
+com.ibm.ws.security.auth.data=io.openliberty.security.auth.internal.data,,"Security Auth Data APIs","Security Auth Data APIs; Jakarta Enabled"
```

Comment:
These descriptions seem backwards.  The APIs are the package that has api in the package.